src/sdk/python/rtdip_sdk/pipelines/data_wranglers/spark/data_quality/prediction/arima.py

```diff
@@ -98,20 +105,26 @@ def __init__(
         concentrate_scale: bool = False,
         trend_offset: int = 1,
         missing: str = "None",
+        arima_auto: bool = False,
     ) -> None:
         if not column_name in past_data.columns:
             raise ValueError("{} not found in the DataFrame.".format(column_name))
 
-        self.df = past_data.toPandas()
+        self.df = past_data
         self.spark_session = past_data.sparkSession
         self.column_to_predict = column_name
         self.rows_to_predict = number_of_data_points_to_predict
         self.rows_to_analyze = number_of_data_points_to_analyze or past_data.count()
-        input_data = (
-            self.df.loc[:, column_name]
-            .sort_index(ascending=True)
-            .tail(number_of_data_points_to_analyze)
-        )
+        self.arima_auto = arima_auto
```

Comment:
Currently it's possible to both put arima_auto = true and set orders in the same component. How to untangle this, to make it impossible to input both variables at the same time? Maybe a own component?

Reply:
I see your point but this is intended. I'd strongly vote that this is up to the components documentation and the users capabilities to utilize the different functionalities of the component. The decision to spin off a new component can later be made by the POs in a next step.